src/views/RouteTutkinnonosat.vue

```diff
@@ -130,6 +176,42 @@ export default class RouteTutkinnonosat extends Vue {
     await this.toteutussuunnitelmaStore.initNavigation();
   }
 
+  selectAllRows() {
+    this.valitseKaikki = !this.valitseKaikki;
+    if (this.valitseKaikki) {
+      this.selectedTutkinnonosat = [
+        ...this.selectedTutkinnonosat,
```

Comment:
Onko mahdollista, että tulee tuplana rivejä this.selectedTutkinnon osaan jos ensin valitsee muutaman ja sen jälkeen valitsee kaikki? Pitäisikö tästä ottaa pois ...this.selectedTutkinnonosat?